README.md

```diff
@@ -1,35 +1,63 @@
-[![NodeCG Logo](http://i.imgur.com/8PjMIL4.png)](http://nodecg.com/)
+# NodeCG
 
-NodeCG is a broadcast graphics framework and application.
-It is primarily aimed at [Twitch](http://twitch.tv) broadcasters using 
-[Open Broadcaster Software](https://obsproject.com/), but is usable in any environment that can render HTML,
-including CasparCG. NodeCG is based on the overlay system developed for the 
-[Tip of the Hats 2014](https://www.youtube.com/watch?v=x9PzBHgN29U) charity event.
+[![NodeCG](media/splash.png)](https://nodecg.com/)
 
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/nodecg/nodecg)
 [![Build Status](https://travis-ci.org/nodecg/nodecg.svg?branch=master)](https://travis-ci.org/nodecg/nodecg)
 [![Coverage Status](https://coveralls.io/repos/nodecg/nodecg/badge.svg?branch=master&service=github)](https://coveralls.io/github/nodecg/nodecg?branch=master)
 [![Greenkeeper badge](https://badges.greenkeeper.io/nodecg/nodecg.svg)](https://greenkeeper.io/)
 
-Looking for a list of NodeCG bundles and resources? Check out [awesome-nodecg](https://github.com/nodecg/awesome-nodecg/blob/master/README.md).
+> Create broadcast graphics using Node.js and a browser
 
-Have questions about NodeCG, or just want to say 'hi'? Join our Gitter chatroom!  
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/nodecg/nodecg)
+NodeCG is a broadcast graphics framework and application. It enables you to write complex, dynamic broadcast graphics
+using the web platform. NodeCG has no graphics or drawing primitives of its own. Instead, NodeCG provides
+a structure for your code and an API to facilitate moving data between the dashboard, the server, and your graphics.
+It makes no assumptions about how to best code a graphic, and gives you complete freedom to use whatever libraries, 
+frameworks, tools, and methodologies you want. As such, NodeCG graphics can be rendered in any environment that
+can render HTML, including:
+- [OBS Studio](https://obsproject.com/) (via the [obs-browser](https://github.com/kc5nra/obs-browser) plugin)
+- [vMix](http://www.vmix.com/)
+
+> Don't see your preferred streaming software on this list? NodeCG graphics require Chrome 49 or newer.
```

Comment:
Predicting questions about obs classic/xsplit/casper support.
What're the chances babel/polyfill could resolve that at some point in the future. classic is eol, but the other two are pretty relevant.

Reply:
The main browser feature that NodeCG now uses is [ES6 Proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy), which cannot be polyfilled nor transpiled for. They are not possible to replicate in userspace JavaScript code -- they have to be implemented natively in the browser. ES Proxies were added to Chrome in version 49, so modern versions of NodeCG cannot run on older builds of Chrome.

XSplit actually does now support NodeCG graphics, as it currently uses a build of CEF based on Chrome 57. I will update the readme.